### PR TITLE
docs: relpath

### PR DESCRIPTION
**Pull request by peintnermax**

### CODE REVIEW

File: docs/docs/legal/terms-of-service.md [hunk 1]
```diff
@@ -14,20 +15,20 @@ The terms of service (**"TOS"**) outlined in this document establish the most im
 
 This Agreement has the following appendices. When you enter the Agreement with us, you accept these agreements.
 
-* [**Data Processing Agreement**](data-processing-agreement) - How we process personal data on behalf of you
-* [**Service Level Description**](service-level-description) - What service levels do we guarantee you
-* [**Support Service Descriptions**](support-services) - How we provide support services to you
+- [**Data Processing Agreement**](./data-processing-agreement) - How we process personal data on behalf of you
+- [**Service Level Description**](./service-level-description) - What service levels do we guarantee you
+- [**Support Service Descriptions**](./support-services) - How we provide support services to you
```
Comment (fforootd) [inline docs/docs/legal/terms-of-service.md:20]:
```suggestion
- [**Data Processing Agreement**](../data-processing-agreement) - How we process personal data on behalf of you
- [**Service Level Description**](../service-level-description) - What service levels do we guarantee you
- [**Support Service Descriptions**](../support-services) - How we provide support services to you
```

File: docs/docs/legal/terms-of-service.md [hunk 2]
```diff
@@ -14,20 +15,20 @@ The terms of service (**"TOS"**) outlined in this document establish the most im
...
 
 The following policies complement the TOS. When accepting the TOS, you accept these policies.
 
-* [**Privacy Policy**](privacy-policy) - How we process personal data on our websites and products
-* [**Acceptable Use Policy**](acceptable-use-policy) - What we understand as acceptable and fair use of our Services
-* [**Rate Limit Policy**](rate-limit-policy) - How we avoid overloads of our services
+- [**Privacy Policy**](./privacy-policy) - How we process personal data on our websites and products
+- [**Acceptable Use Policy**](./acceptable-use-policy) - What we understand as acceptable and fair use of our Services
+- [**Rate Limit Policy**](./rate-limit-policy) - How we avoid overloads of our services
```
Comment (fforootd) [inline docs/docs/legal/terms-of-service.md:26]:
```suggestion
- [**Privacy Policy**](../privacy-policy) - How we process personal data on our websites and products
- [**Acceptable Use Policy**](../acceptable-use-policy) - What we understand as acceptable and fair use of our Services
- [**Rate Limit Policy**](../rate-limit-policy) - How we avoid overloads of our services
```

File: docs/docs/legal/terms-of-service.md [hunk 3]
```diff
@@ -14,20 +15,20 @@ The terms of service (**"TOS"**) outlined in this document establish the most im
...
 
 This Agreement is extended with additional terms, in case your Subscription requires a Dedicated Instance. When you enter the Agreement with us, you accept these additional agreements.
 
-* [**Dedicated Instance Terms**](terms-of-service-dedicated) - How we provide our services for a dedicated instance
-* [**Annex to the Dedicated Instance Terms**](dedicated-instance-annex) - Describe differences of services and guarantees with different providers
+- [**Dedicated Instance Terms**](./terms-of-service-dedicated) - How we provide our services for a dedicated instance
+- [**Annex to the Dedicated Instance Terms**](./dedicated-instance-annex) - Describe differences of services and guarantees with different providers
```
Comment (fforootd) [inline docs/docs/legal/terms-of-service.md:31]:
```suggestion
- [**Dedicated Instance Terms**](../terms-of-service-dedicated) - How we provide our services for a dedicated instance
- [**Annex to the Dedicated Instance Terms**](../dedicated-instance-annex) - Describe differences of services and guarantees with different providers
```

**Comment by mffap** [comment]
@peintnermax @fforootd we would need to replace all the links across docs, don't we? This PR does not seem to cover even the rest of the legal page. Are users currently re-directed to the v2 docs, if they click on a link?

**Comment by fforootd** [comment]
I will close this since it is for the old docs which we will deprecate anyway

**Pull request closed without merging**